src/API/QueryFieldLocationService.php

```diff
@@ -0,0 +1,18 @@
+<?php
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+
+namespace EzSystems\EzPlatformQueryFieldType\API;
```

Comment:
Usually it is:
```suggestion

/**
 * @copyright Copyright (C) eZ Systems AS. All rights reserved.
 * @license For full copyright and license information view LICENSE file distributed with this source code.
 */
declare(strict_types=1);

namespace EzSystems\EzPlatformQueryFieldType\API;
```
But I see PHP CS Fixer got you covered there ;)

Reply:
As a matter of fact, I have executed composer install a few minutes ago to execute it and run tests ;)